kitchen.yml

```diff
@@ -9,3 +9,9 @@ suites:
   - name: homes
     run_list:
       - recipe[osl-nfs::homes]
+
+provisioner:
+  name: chef_solo
+  #enforce_idempotency: true
+  multiple_converge: 2
+  deprecations_as_errors: true
```

Comment:
I prefer having provisioner settings at or near the top. Can you please do that?